test-app/.template-lintrc.js

```diff
@@ -38,5 +22,11 @@ module.exports = {
         'require-presentational-children': 'warn',
       },
     },
+    {
+      files: ['tests/**/*.(gjs|gts)'],
+      rules: {
+        prettier: 'off',
+      },
+    },
```

Comment:
Why do we disable Prettier for `.gjs` and `.gts` files? Are we running Prettier for the `<template>` tags with ESLint now?